py/loqui/socket_watcher.pyx

```diff
@@ -9,7 +9,7 @@ cdef class SocketWatcher:
         self.waiter = Waiter()
         self.waiter_get = self.waiter.get
         self.waiter_clear = self.waiter.clear
-        self.waiter_switch = self.waiter.switch
+        self.waiter_switch = lambda: self.waiter.switch(None)
```

Comment:
Waiter.switch used to look like this, but the default value was removed in 1.3, so passing in None here is not a change in behavior.
```
def switch(self, value=None)
```

Reply:
Ran into the exact the same problem. Hope this can be merged.